### ISSUE TITLE
Figure out how to manage our Git workflow. 

### ISSUE THREAD

**Issue by guy-jerome** [grup5/Frontend-Capstone#4]
We need to get a system in place for implementing the Git workflow. Here is an article that might be useful:
https://nvie.com/posts/a-successful-git-branching-model/

**Comment by guy-jerome** [comment]
We are working and committing to the Dev branch. Each member works on their feature branch and once per day, the dev branch is merged into the main branch.
